core/components/commerce_timeslots/src/Modules/TimeSlots.php

```diff
@@ -60,8 +67,7 @@ public function initialize(EventDispatcher $dispatcher)
         $path = $root . 'src/Services/Scheduler/crontime';
         if (file_exists($path)) {
             // Runs at midnight each night.
-            $this->commerce->scheduler()->repeat([$this, 'populateDailySlots'], Interval::daily(0, 0));
-            //$this->commerce->scheduler()->repeat([$this, 'populateDailySlots'], Interval::hourly('*'));
+            $this->commerce->scheduler()->repeat([$this, 'populateDailySlots'], Interval::weekly('*', 0,0));
```

Comment:
Huh, this is an interesting change... is there a problem with the daily interval?

Reply:
Nothing wrong here (besides me forgetting to change it back)!